redfish/provider_test.go

```diff
@@ -0,0 +1,34 @@
+package redfish
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"os"
+)
+
+var testAccProviders map[string]*schema.Provider
+var testAccProvider *schema.Provider
+
+// TestingServerCredentials Struct used to store the credentials we pass for testing. This allows us to pass testing
+// credentials via environment variables instead of having them hard coded
+type TestingServerCredentials struct {
+	Username string
+	Password string
+	Endpoint string
+	Insecure bool
+}
+
+var creds = TestingServerCredentials{
```

Comment:
Hi @grantcurell,

Just to make it compliant with the other two vars, I think you should declare the *creds* var under *testAccProviders* and *testAccProvider* as an empty var, and then init it within the *init()* function, as those two vars are:
Something like:
~~~
var testAccProviders map[string]*schema.Provider
var testAccProvider *schema.Provider
var creds TestingServerCredentials

func init() {

	testAccProvider = Provider()
	testAccProviders = map[string]*schema.Provider{
		"redfish": testAccProvider,
	}
        
        creds = TestingServerCredentials{
                           Username: os.Getenv("TF_TESTING_USERNAME"),
	                   Password: os.Getenv("TF_TESTING_PASSWORD"),
	                   Endpoint: os.Getenv("TF_TESTING_ENDPOINT"),
	                   Insecure: false,
                   }
}
~~~

Everyting else look good to me 😄

Reply:
@mikeletux I tried that - there's something about the scope that doesn't work:

![image](https://user-images.githubusercontent.com/35169230/117666643-11ec0280-b172-11eb-9a51-1a8dbf5e9f10.png)